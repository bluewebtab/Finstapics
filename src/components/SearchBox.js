import React from 'react';

const SearchBox = ({searchChange}) => {


    return (
        <div className="container">
            <div className = 'pa2 search-box'>
                <div className="content">
                    <input
                        aria-label='search images'
                        className="pa3"
                        type='search'
                        placeholder='search images'
                        onChange={searchChange}
                    />
                </div>
            </div>
        </div>
    )
}

export default SearchBox;