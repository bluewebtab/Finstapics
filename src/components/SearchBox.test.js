import {shallow, mount, render} from 'enzyme';
import React from 'react';
import SearchBox from './SearchBox';

let wrapper;

beforeEach(() => {
   const mockProps = {
    onRequestImages: jest.fn()
   }

   wrapper = shallow(<SearchBox {...mockProps}/>)
})

it('renders SearchBox without crashing', () => {
    expect(wrapper).toMatchSnapshot();
    
})