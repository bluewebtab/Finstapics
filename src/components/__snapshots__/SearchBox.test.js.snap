// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MainPage without crashing 1`] = `undefined`;

exports[`renders SearchBox without crashing 1`] = `ShallowWrapper {}`;
